### ISSUE TITLE
Unable to change the log read size

### ISSUE THREAD

**Issue by cztbg0** [mapwatch/mapwatch#382]
As the title states, changing the size of the log parse from 20MB doesn't do anything. This is relevant in regards to league-long data, as my current league log is 28MB long and we're only about half way through the league.

**Comment by cztbg0** [comment]
Bumping this, it seems like poe's logs have increased in size in the past few leagues. Back when I made the original post it took about half a league before I noticed that the 20MB limit would bump against the league start. 


![image](https://user-images.githubusercontent.com/96273362/215661779-19a7d3c7-8f93-4b0b-9572-087093eef96c.png)
My client.txt file is 59MB and dates back to June 2022. The screenshot above shows that the latest 20MB in client.txt only cover the time period for the last 2 weeks, or around 838 maps. Changing the log read size from 20MB to 30 or 50 does not actually cause mapwatch to process more data, the status text on the bottom still reads "Processed 20MB in x seconds".

I tried using an alternate client.txt file that removed all logs from before 2022-12-09 (the day the current league launched), but that did not resolve the issue.
